src/components/breadcrumbs/crumb/crumb.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import Crumb from "./crumb.component";
+
+const LINK_TEXT = "Link text";
+
+describe("Crumb", () => {
+  it("passes href to the anchor element isCurrent is false", () => {
+    render(<Crumb href="foo" text={LINK_TEXT} />);
+
+    const link = screen.getByText(LINK_TEXT);
+    expect(link).toBeInTheDocument();
+    expect(link?.closest("a")).toHaveAttribute("href", "foo");
+  });
+
+  it("does not pass href to the anchor element when isCurrent is true", () => {
+    render(<Crumb href="foo" text={LINK_TEXT} isCurrent />);
+
+    const anchor = screen.getByText(LINK_TEXT).closest("a");
+    expect(anchor).not.toHaveAttribute("href", "foo");
+
+    expect(anchor).toHaveStyle({
+      color: "var(--colorsYin055)",
+      textDecoration: "none",
+      cursor: "text",
+    });
+  });
+
+  it("calls the onClick callback if one is passed and isCurrent is false", () => {
```

Comment:
Personally I would opt to use React Testing Library's `userEvent` helper library instead of `fireEvent`, since the helpers it provides simulate user interactions more closely rather than simulating a dispatched DOM event, giving us greater security that our tests mimic how a user would use the component. 

However, I'm happy to accept these two tests as they are here, since the action here isn't complicated